bin/dist-diff.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/sh
+set -e
```

Comment:
Little heads up that, I've udpdated this part to match the rest of the scripts in `bin`.